src/session.ts

```diff
@@ -320,6 +322,18 @@ export class SessionManager implements Middleware {
             return resolvedCodeLens;
     }
 
+    // During preview, populate a new setting value but not remove the old value.
+    // When the PowerShell extension releases the RTM version, then the old value can be safely removed.
+    private async migrateWhitespaceAroundPipeSetting() {
```

Comment:
I'd rather defer it to when it is needed as it might be a YAGNI.
Or maybe I should create an NPM package for it so that I can break the Internet later one 😂
https://www.zdnet.com/article/another-one-line-npm-package-breaks-the-javascript-ecosystem/